wire-ios/Wire-iOS/Sources/UserInterface/Availability/AvailabilityStringBuilder.swift

```diff
@@ -58,7 +58,14 @@ final class AvailabilityStringBuilder: NSObject {
         guard let textColor = color,
               let iconColor = iconColor else { return nil }
         let icon = AvailabilityStringBuilder.icon(for: availability, with: iconColor, and: fontSize)
-        let attributedText = IconStringsBuilder.iconString(with: icon, title: title, interactive: false, color: textColor)
+        let attributedText = IconStringsBuilder.iconString(
+            leadingIcons: [icon].compactMap(\.self),
+            title: title,
+            trailingIcons: [],
+            interactive: false,
+            color: textColor,
+            titleFont: nil
```

Comment:
`titleFont` has a default nil value, you don't need to add it.

Reply:
Reverted.